video/cloud-client/analyze/video_detect_logo.py

```diff
@@ -14,16 +14,16 @@
 
 import io
 
-# [START video_detect_logo_beta]
+# [START video_detect_logo]
```

Comment:
So we try not to do this as it will break docs. Prefer to create a new GA file rather than rename beta files.

Reply:
Ah dang good point. Do we then follow up and delete the beta files?